### DISCUSSION TITLE
HAProxy 2.8.0

### DISCUSSION THREAD

**Discussion by lordbuddha** [amazonlinux/amazon-linux-2022#443] [General]
Any chance that haproxy will be updated from 2.8.0 to a more recent maintenance release (2.8.3) on AL2023

The current 2.8.0 is a big security headache for us as HTTP POST handling in a LUA Applet is broken, but is fixed in later maintenance releases.

**Comment by lordbuddha** [comment]
https://www.mail-archive.com/haproxy@formilux.org/msg43718.html  describes the fix.

    In the lua, receive functions for HTTP applets were fixed to properly
    detect the end of messages. This was broken during the stream-connector
    refactoring. It was possible to hang on received because the wrong SC
    was tested. Still in the lua, the get_stat() function was fixed. It
    stopped working because the "proto" field definition was missing.

**Comment by alexey-tsvetnov** [comment]
The current HAProxy's version is 3.0.5

```
$ cat /etc/amazon-linux-release
Amazon Linux release 2023.10.20260302 (Amazon Linux)

$ dnf info haproxy
Name         : haproxy
Version      : 3.0.5
Release      : 1.amzn2023.0.1
```
